clients/admin-ui/src/features/config-wizard/ScanResultsForm.tsx

```diff
@@ -33,26 +36,33 @@ const ALL_COLUMNS: ColumnMetadata[] = [
 const ScanResultsForm = () => {
   const systems = useAppSelector(selectSystemsForReview);
   const dispatch = useAppDispatch();
+  const router = useRouter();
   const {
     isOpen: isWarningOpen,
     onOpen: onWarningOpen,
     onClose: onWarningClose,
   } = useDisclosure();
+  const [upsertSystems] = useUpsertSystemsMutation();
   const [selectedSystems, setSelectedSystems] = useState<System[]>(systems);
+  const features = useFeatures();
   const [selectedColumns, setSelectedColumns] =
     useState<ColumnMetadata[]>(ALL_COLUMNS);
 
+  const createSystems = async () => {
+    await upsertSystems(selectedSystems);
```

Comment:
we should get the result and handle errors, something like this pattern: https://github.com/ethyca/fides/blob/main/clients/admin-ui/src/features/config-wizard/AuthenticateAwsForm.tsx/#L103-L116

and then we'd probably want a toast or something, not sure if the designs have something like that

Reply:
I think the toasts (or at least, response messages feedback to the user) are going to be handled by [this issue](https://github.com/orgs/ethyca/projects/17/views/1?filterQuery=is%3Aissue+sprint%3A%40current+assignee%3ALKCSmith) when I get to it next, but let me know if you think otherwise! I'll add the other things though.

Reply:
@allisonking Actually, are you okay with me handling the success/responses from this upsert in that PR related to that issue, since the config wizard is hidden for now? Toasts aren't in the designs, and I think that issue covers your concerns here when I get to it later today.